quickwit-search/src/tests.rs

```diff
@@ -377,6 +377,70 @@ async fn test_single_node_sorting_with_query() -> anyhow::Result<()> {
     Ok(())
 }
 
+// Sorry Evance for stealing your test
```

Comment:
You know what they say about copy-pasting code, right? :)
Now, you've earned the right to write a helper function to run those two tests.